src/app/shared/data-input/data-input.component.html

```diff
@@ -132,6 +132,20 @@
                    (dataBlur)="onDataBlur()">
   </user-data-input>
 
+  <ng-container *ngSwitchCase="constraintType.Text">
+    <rich-text-data-input *ngIf="!configuration.fromQuery; else textDataInput"
```

Comment:
How is html handled in query?

Reply:
This requires a later change on backend.